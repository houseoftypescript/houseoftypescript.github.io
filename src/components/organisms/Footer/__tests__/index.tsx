import { render } from '@testing-library/react';
import { Footer } from '../index';

describe('Footer', () => {
  it('renders correctly', () => {
    const button = render(<Footer title="" socialLinks={[]}></Footer>);
    expect(button.container).toMatchSnapshot();
  });
});
