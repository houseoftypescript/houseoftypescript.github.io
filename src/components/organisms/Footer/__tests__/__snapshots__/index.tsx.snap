// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <footer
    class="py-8 border-t md:border-none"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="flex items-center justify-between"
      >
        <div
          class="font-medium font-lg"
        >
          Â© 
          2023
           
        </div>
        <div
          class="flex items-center gap-2 md:gap-4"
        />
      </div>
    </div>
  </footer>
</div>
`;
